sources/packages/backend/libs/sims-db/src/entities/application-offering-change-request.model.ts

```diff
@@ -15,6 +15,7 @@ import { Note } from "./note.model";
 import { ApplicationOfferingChangeRequestStatus } from "./application-offering-change-request-status.type";
 
 export const REASON_MAX_LENGTH = 500;
+export const NOTE_MAX_LENGTH = 500;
```

Comment:
![image](https://github.com/bcgov/SIMS/assets/54600590/902e187d-918c-4667-980c-1ad65dd094c2)

the notes max length is already available